packages/editor/src/components/properties/NameInputGroup.tsx

```diff
@@ -18,58 +23,63 @@ const StyledNameInputGroup = (styled as any)(InputGroup)`
   }
 `
 
-type Types = {
-  node: any
-}
-
 /**
  * NameInputGroup is used to render input group PropertiesPanelContainer.
  *
  * @author Robert Long
  * @type {class component}
  */
-export const NameInputGroup = (props: Types) => {
-  let [name, setName] = useState(props.node.name)
-  let [focusedNode, setFocusedNode] = useState(null)
+export const NameInputGroup: EditorComponentType = (props) => {
+  const nodeName = getComponent(props.node.entity, NameComponent).name
+
+  let [name, setName] = useState(nodeName)
+  let [focusedNode, setFocusedNode] = useState<EntityTreeNode>()
```

Comment:
Its better to use `const` for react component states

Reply:
Fixed.